src/node/BrowserFetcher.ts

```diff
@@ -112,10 +112,12 @@ function handleArm64(): void {
     if (stats === undefined) {
       fs.stat('/usr/bin/chromium', function (err, stats) {
         if (stats === undefined) {
-          console.error(`The chromium binary is not available for arm64.`);
-          console.error(`If you are on Ubuntu, you can install with: `);
-          console.error(`\n sudo apt install chromium\n`);
-          console.error(`\n sudo apt install chromium-browser\n`);
+          console.error(
+            'The chromium binary is not available for arm64.' +
+              'If you are on Ubuntu, you can install with: ' +
+              '\n sudo apt install chromium\n' +
+              '\n sudo apt install chromium-browser\n'
+          );
```

Comment:
This is not equivalent. Please revert this part of the change.

Reply:
It is equivalent now.